DeviceAdministration/Web/WebApiControllers/TelemetryApiController.cs

```diff
@@ -250,11 +250,18 @@ public class TelemetryApiController : WebApiControllerBase
                         historyItems.AddRange(data);
 
                         List<dynamic> devices = await LoadAllDevicesAsync();
-                        devices.ForEach(device => TypeMapper.Get().map<DeviceND>(device));
+                        List<DeviceND> resultND = new List<DeviceND>();
```

Comment:
And it's a preference but I usually initialize Lists using IList or IEnumerable. Explained why here: **http://stackoverflow.com/questions/17170/when-to-use-ilist-and-when-to-use-list**